### PR TITLE
Feature/1432 4xl avatar

### PR DESCRIPTION
**Pull request by anna-lach**

### CODE REVIEW

File: .changeset/tricky-buttons-ring.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+'@sl-design-system/kampus': minor
+'@sl-design-system/magister': minor
+'@sl-design-system/max': minor
+'@sl-design-system/my-digital-book': minor
+'@sl-design-system/neon': minor
+'@sl-design-system/sanoma-learning': minor
+'@sl-design-system/teas': minor
+---
+
+Added tokens for new avatar size: `4xl`
```
Comment (jpzwarte) [inline .changeset/tricky-buttons-ring.md:16]: Themes can be just `patch`

**Review by jpzwarte — APPROVED** [review]

File: packages/locales/src/nl.ts [hunk 2]
```diff
@@ -37,8 +35,8 @@ export const templates = {
   sd244b51f6d7de8e2: 'Voer een waarde in volgens het gevraagde patroon.',
   sd711c88b851db6c9: 'De volgende velden zijn incorrect:',
   sde6884478b7cc818: str`Voer tenminste ${0} karakters in (je hebt op dit momoment ${1} karakter${2}).`,
-  sea4f08110bb8f15d: 'verwijderen',
   see63aaad45b1b116: 'status',
   sf1ec4acb8d744ed9: 'Mededeling',
-  sf677da98fa27f9b6: 'Meer links'
+  sf677da98fa27f9b6: 'Meer links',
+  sf75c5ddbac2b60f9: str`Showing ${0} out of ${1} items`
```
Comment (Diaan) [inline packages/locales/src/nl.ts:41]: I don't think these changes are supposed to be there. I got them as well...
has there been some merging in the localisation file that generates this file?

File: website/src/_data/tokens/base.json [hunk 3]
```diff
@@ -1983,7 +2021,7 @@
     },
     "focusring": {
       "default": {
-        "value": "{borderRadius.default} - 1px",
+        "value": "{borderRadius.default}+1",
```
Comment (Diaan) [inline website/src/_data/tokens/base.json:2024]: doesn't this need spaces around the `+` (and a unit behind the `1`)

File: packages/locales/src/nl.ts (hunk 2 above)
Comment (anna-lach) [reply packages/locales/src/nl.ts]: There were sth wrong with translations, I've just fixed/changed them.

File: packages/locales/src/nl.ts [hunk 4]
```diff
@@ -35,8 +36,9 @@ export const templates = {
   sd244b51f6d7de8e2: 'Voer een waarde in volgens het gevraagde patroon.',
   sd711c88b851db6c9: 'De volgende velden zijn incorrect:',
   sde6884478b7cc818: str`Voer tenminste ${0} karakters in (je hebt op dit momoment ${1} karakter${2}).`,
+  sebd54694b2f83547: str`verwijderen '${0}'`,
```
Comment (Diaan) [inline packages/locales/src/nl.ts:39]:
```suggestion
  sebd54694b2f83547: str`verwijder '${0}'`,
```

File: packages/locales/src/nl.ts [hunk 5]
```diff
@@ -35,8 +36,9 @@ export const templates = {
...
   see63aaad45b1b116: 'status',
   sf1ec4acb8d744ed9: 'Mededeling',
   sf677da98fa27f9b6: 'Meer links',
-  sf75c5ddbac2b60f9: str`Showing ${0} out of ${1} items`
+  sf75c5ddbac2b60f9: str`Met ${0} uit ${1} items`
```
Comment (Diaan) [inline packages/locales/src/nl.ts:43]:
```suggestion
  sf75c5ddbac2b60f9: str`${0} van ${1} items zichtbaar`
```

**Pull request merged**